script/script.js

```diff
@@ -446,7 +446,7 @@ function loop(unadjusted, off = 0) {
       ? (ms / 1000) * game.shiftAuto.toNumber()
       : 0;
   if (game.cAutoLoop.shift >= 1) {
-    if (!(game.advAutoShift==1&&game.factorShifts==5&&inChal(8))) factorShift();
+    if (!(game.advAutoShift==1&&game.factorShifts>=5&&inChal(8))) factorShift();
```

Comment:
This row is fix the number doesn't formatted in late-game

Reply:
This isn't a fix.. it already auto shifts to FS7 in C8. `if NOT (advautoshift = on AND factorshift = 5) factorshift()`. so if you manually shift from 5 to 6 it will shift to 7 since it's no longer at 5.
this would cause it to stop working

Reply:
if you point is that you don't want it to i ask you why?

Reply:
Sorry, it is because wording case a bug not running case a bug